Makefile

```diff
@@ -27,8 +27,9 @@ GOARCH ?= $(shell go env GOARCH)
 
 # Default os-arch combination to build
 XC_OS ?= darwin freebsd linux netbsd openbsd solaris windows
-XC_ARCH ?= 386 amd64 arm
-XC_EXCLUDE ?= darwin/arm solaris/386 solaris/arm windows/arm
+XC_ARCH ?= 386 amd64 arm arm64
+# XC_EXCLUDE "arm64" entries match/exclude both arm and arm64
```

Comment:
Making sure I understand this comment, I'm understanding it as meaning the text "arm64" below means to exclude arm and arm64 for a given OS. E.G. `darwin/arm64` means exclude `darwin/arm64` as well as `darwin/arm`. Am I understanding that right?

Reply:
Yep. The makefile is using substring matching to exclude OS/Arch combos (this predates me, but works ok), matching the OS/Arch against each entry in the Exclude list. So when it checks `darwin/arm' it matches the first part of 'darwin/arm64' and is excluded.

This caught me at first and so the comment to help clarify. There might be a better way to do it, but I wasn't looking to rewrite the Makefile at this point.